azure-spring-apps-enterprise/workshops/azure-spring-apps-enterprise/ai/02-prepare-azure-openai/README.md

```diff
@@ -8,6 +8,7 @@ It's very important that your Azure subscription has been granted access to Azur
 1. Run the following command to create an Azure OpenAI resource in the the resource group.
 
    ```bash
+   az upgrade -y
```

Comment:
nit: Since this command will upgrade Azure CLI and the extensions, shall we also add a step to let user/trainee to check if the version is good?

Such as `az version`, and make sure "azure-cli" and "azure-cli-core" version is great than a version, such as "2.47.0"?

Reply:
It should be fix. I remember perquisition already tells user how to check az version